### PR TITLE
Server/install bug fixes

### PR DESCRIPTION
**Pull request by rosieyohannan**
Fixes to server 3.2 installation docs post release.

### CODE REVIEW

File: jekyll/_cci2/server-3-install-build-services.adoc [hunk 1]
```diff
@@ -20,7 +22,7 @@ toc::[]
 ==== Overview 
 Output processor is responsible for handling the output from Nomad clients. It is a key service to scale if you find your system slowing down. We recommend increasing the output processor replica set to scale the service up to meet demand. 
 
-Access the KOTS admin console. Get to the KOTS admin console by running the following, substituting your namespace: `kubectl kots admin-console -n <namespace>`
+Access the KOTS admin console. Get to the KOTS admin console by running the following, substituting your namespace: `kubectl kots admin-console -n <YOUR_CIRCLECI_NAMESPACE>`
 
 Locate and enter the following in Settings:
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-build-services.adoc:28]: Should this be `kubectl get service output-processor --namespace=<YOUR_CIRCLECI_NAMESPACE>`

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-build-services.adoc:28]: @circleci/server

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 2]
```diff
@@ -155,8 +155,6 @@ By default, CircleCI server will create self-signed certificates to get you star
 
 ==== Google Cloud DNS
 
-NOTE: In the following steps replace any items credentials displayed between `< >` with your details.
-
 . If you host your DNS on Google Cloud you will need the *certbot-dns-google* plugin installed. You can install the plugin with the following command: 
 +
 ```bash
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:160]: @circleci/server  Below where we have `<CIRCLECI_SERVER_DOMAIN>` is that the same as `<YOUR_CIRCLECI_NAMESPACE>` that we use in most places? I just want to keep this consistent if possible.

**Review by ryanwohara — APPROVED** [review]

File: jekyll/_cci2/server-3-install-build-services.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-build-services.adoc:28]: @ryanwohara just checking on these two comments if the change I suggest would be correct?

Comment (ryanwohara) [reply jekyll/_cci2/server-3-install-build-services.adoc:28]: Agreed it should not just be circleci-server - I personally use rohara-dev!

File: jekyll/_cci2/server-3-install-prerequisites.adoc (hunk 2 above)
Comment (ryanwohara) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:160]: No, server domain is the domain that is used to access the application

File: jekyll/_cci2/server-3-install-build-services.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-build-services.adoc:28]: Thanks!

File: jekyll/_cci2/server-3-install-prerequisites.adoc (hunk 2 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:160]: I see, thanks!

**Pull request merged**